Cargo.toml

```diff
@@ -4,11 +4,13 @@ members = [
     "aead",
     "cipher",
     "crypto",
-    "crypto-common",
-    "digest",
     "elliptic-curve",
     "kem",
     "signature",
     "signature/async",
     "universal-hash",
 ]
+exclude = [
```

Comment:
Aha! So that's how you do this properly

Reply:
Unfortunately, I am not sure it will work as we would like. Running `cargo +1.41.0 build` in the `digest` folder results in the following warning:
```
warning: unused manifest key: workspace.resolver
```

To be safe I simply will republish `digest` using 1.41 toolchain.